inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -25,22 +26,22 @@ class SourceParameters:
 
 @dataclass(kw_only=True, frozen=True)
 class ApplicationAddSourceParameters:
-    gpg_key_uri: str
-    gpg_key_name: str
-    file_name: str
-    sources: list[str] = field(default_factory=lambda: [])
+    source_list_file_name: str
+    sources: List[str] = field(default_factory=list)
```

Comment:
**Best Practice**
**Suggestion:** To avoid mutable default arguments and ensure that each instance of ApplicationAddSourceParameters has its own list, use a lambda function for the default factory.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    @dataclass(kw_only=True, frozen=True)
    class ApplicationAddSourceParameters:
        sources: List[str] = field(default_factory=lambda: [])
```